step.yml

```diff
@@ -0,0 +1,49 @@
+
+title: Create install page QR code
+summary: Generate an URL for a QR code which points to the install page.
+description: Generate an URL for a QR code which points to the install page.
+website: https://github.com/bitrise-steplib/steps-create-install-page-qr-code
+source_code_url: https://github.com/bitrise-steplib/steps-create-install-page-qr-code
+support_url: https://github.com/bitrise-steplib/steps-create-install-page-qr-code/issues
+host_os_tags:
+  - osx-10.10
+  - ubuntu-16.04
+
+type_tags:
+  - deploy
+
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: true
+
+toolkit:
+  go:
+    package_name: github.com/bitrise-steplib/steps-create-install-page-qr-code
+
+inputs:
+  - public_install_page_url:
+    opts:
+      title: "Public Install Page URL"
+      summary: "The full path to the application file (.app or .apk)"
+      description: "The full path to the application file (.app or .apk)"
+      is_expand: true
```

Comment:
no need to define options with default value: https://github.com/bitrise-io/envman/blob/master/models/models_methods.go#L20